en/xml/fo/topic.graphic-pageheader.xml

```diff
@@ -0,0 +1,259 @@
+                                    <tag>graphics</tag> .</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.blank">
+                            <para>Here you can place an banner like
+                                    '<emphasis>This page is intentionally left
+                                    blank'</emphasis> or so.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.center">
+                            <para>This is a good place for the chapter
+                                title.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.right">
+                            <para>This is the place where we want to place our
+                                graphic. The graphic should be visible on all
+                                pages on the right side.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.height">
+                            <para>To avoid conflicts with the body text area,
+                                we limit its height to 1 inch. When you
+                                specify only one parameter like
+                                    <tag>height</tag>or<tag>width</tag>, your
+                                graphic will be scaled with the <link
+                                    xlink:href="#li.fo.external-graphics"
+                                    >correct aspect ratio</link>.</para>
+                        </callout>
+                        <callout
+                            arearefs="co.topic.graphic-pageheader.attribute">
+                            <para>The filename is passed on the <tag
+                                    class="attribute">src</tag> attribute.
+                                    <emphasis>src</emphasis>.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.template">
+                            <para>The template which is called to import and
+                                place the graphic.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.file">
+                            <para>Our file defined at the top of our
+                                customization layer.</para>
+                        </callout>
+                    </calloutlist>
+                </example>
+            </step>
+            <step>
+                <para>Use your customization layer to transform your DocBook
+                    document into FO.</para>
+            </step>
+        </procedure>
+    </section>
+    <section role="discussion">
+        <title>Discussion</title>
+        <para>These points will be submitted later</para>
```

Comment:
Here you could describe:

* Put logo(s) inside a different directory and assign it with your new parameter `logo.src.path`.

Reply:
Done.